.github/workflows/main.yml

```diff
@@ -17,11 +17,9 @@ jobs:
           node-version: 18
           cache: 'npm'
       - run: npm ci
-      - run: npm run build
```

Comment:
build is redundant here, now we call it from `prepare` it happens as part of `npm ci`